avocado_vt/test.py

```diff
@@ -150,6 +150,11 @@ def setUp(self):
         except exceptions.TestSkipError:
             self.__exc_info = sys.exc_info()
             raise  # This one has to be raised in setUp
+        except OSError as e:
+            if not self._config.get("vt.omit_data_loss"):
```

Comment:
since we need this solution urgently, if in short term, we can not have complete solution. Maybe we can just output the error message, but never raise exception in any situation. But surely we can open one issue to track limitation, and further work out complete solution

Reply:
I have just sent an update via `force-push` which keeps the messages, but it should stop raising the exceptions.